packages/core/src/browser/shell/application-shell.ts

```diff
@@ -619,7 +621,7 @@ export class ApplicationShell extends Widget {
      */
     addWidget(widget: Widget, options: ApplicationShell.WidgetOptions) {
         if (!widget.id) {
-            console.error('Widgets added to the application shell must have a unique id property.');
```

Comment:
using the console is preferred way of login in the frontend, otherwise all logs will end up only on the backend or lost if the connection is lost.

Reply:
Good point; but the console logger is the default. I can extract the warning-logging parts into methods in the application shell, and on can override this behavior. What do you think?

If you still prefer `console.log` instead of the logger, how would you address this comment:

> So, from what I understand, we can and want to get rid of the console usages, at least in the frontend? If so, why re-introduce the comments that silence the console warnings?